How-to-Monitor-Docker-Containers.md

```diff
@@ -55,6 +55,46 @@ ExecStart=/usr/bin/dockerd --containerd=/run/containerd/containerd.sock
 
 My original ExecStart was: `ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock`, note the -H that would cause a duplicate property error.
 
+
+### Method 2 - If you installed docker using snap
+
+Tested in Ubuntu 22
+
+The location of the files is different:  **daemon.json** is in /var/snap/docker/current/config/
+
+`sudo nano /var/snap/docker/current/config/daemon.json`
+
+And this is what the original file looks like
+
+```
+{
+    "log-level":        "error",
+    "storage-driver":   "overlay2"
+}
+```
+
+Change it to 
+
+```
+{
+    "log-level":        "error",
+    "storage-driver":   "overlay2",
+    "hosts": ["unix:///var/run/docker.sock", "tcp://0.0.0.0:2375"]
+}
```

Comment:
see https://stackoverflow.com/questions/40883421/diff-syntax-highlighting-in-github-markdown
```suggestion
```diff
{
    "log-level":        "error",
    "storage-driver":   "overlay2",
+   "hosts": ["unix:///var/run/docker.sock", "tcp://0.0.0.0:2375"]
}